src/tools/roc_send/main.cpp

```diff
@@ -108,25 +114,39 @@ int main(int argc, char** argv) {
     sndio::Config io_config;
     io_config.sample_spec.set_channel_set(sender_config.input_sample_spec.channel_set());
 
-    if (args.packet_length_given) {
-        if (!core::parse_duration(args.packet_length_arg, sender_config.packet_length)) {
-            roc_log(LogError, "invalid --packet-length");
+    if (args.packet_len_given) {
+        if (!core::parse_duration(args.packet_len_arg, sender_config.packet_length)) {
+            roc_log(LogError, "invalid --packet-len");
             return 1;
         }
     }
 
-    if (args.frame_length_given) {
-        if (!core::parse_duration(args.frame_length_arg, io_config.frame_length)) {
-            roc_log(LogError, "invalid --frame-length: bad format");
+    if (!args.max_packet_size_given) {
+        // TODO(gh-608): take size from --packet-encoding instead of assuming 2 bytes per
+        // sample
```

Comment:
Can you please reorder code in roc-send so that max_packet_size_given checks would be grouped together, like you did in roc-recv?

Like:

```cpp
if (max_packet_size_given) {
    ...
} else {
    ...
}
```

It's much easier to follow the code when these "if" and "else" for every option are in one place.

(Sorry that I didn't notice it in previous reviews).